project/standard/webpack.config.js

```diff
@@ -0,0 +1,49 @@
+const path = require("path");
+
+const extractThemesPlugin = require('../../build/themes.js').extractThemesPlugin;
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+const paths = {
+    base: path.join(__dirname, "..", "..", "..", ".."),
+    dist: path.join(__dirname, "..", "..", "..", ".."),
+    framework: path.join(__dirname, "..", "..", "..", "..", "node_modules", "mapstore", "web", "client"),
+    code: [
+        path.join(__dirname, "..", "..", "..", "..", "js"),
+        path.join(__dirname, "..", "..", "..", "..", "node_modules", "mapstore", "web", "client")
+    ]
+};
+
+const mapstoreConfig = require(path.join(__dirname, "..", "..", "..", "..", "package.json")).mapstore || {};
```

Comment:
can we document this new possibility somewhere?

Reply:
@MV88 I updated the PR with a comment